rust-connectors/common/Cargo.toml

```diff
@@ -23,7 +23,9 @@ clap = { version = "3.1", features = ["std", "derive"], default-features = false
 
 anyhow = "1.0.56"
 fluvio-future = { version = "0.4.1", features = ["subscriber"] }
-fluvio = { version = "0.13" }
+fluvio = { git = "https://github.com/infinyon/fluvio" }
```

Comment:
Should use patch dependencies: https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html.

Reply:
Unfortunately, I didn't manage to do it here. Due to different features set between the patch and the original cargo complains and says that the patch is not applied.

Reply:
Careful reading documentation helped :). Fixed.